src/constants.js

```diff
@@ -1 +1,8 @@
 export const SERVER_URL = 'http://localhost:8081';
+
+
+export const SEMESTER_LIST = [
+    {id:0, year:2020, name:'Fall'},
+    {id:1, year:2021, name:'Spring'},
+    {id:2, year:2021, name:'Fall'}
+  ] ;
```

Comment:
"No newline at EOF" likely is not causing issues, but consider revising or reviewing in the future if problems arise.

Reply:
Thanks for the catch.